README.rst

```diff
@@ -108,6 +122,186 @@ to enable source maps in Google Chrome, if you haven't already done so.
+The items can provide a target as a HTML data attribute (**data-showroom-target**) to define the endpoint where the item will fetch its content from.
+To make a title visible in the overlay provide a title as a HTML data attribute  (**data-showroom-title**).
+
+.. code:: HTML
+
+  <ul>
+    <li class="item"
+        data-showroom-target="http://target.com/1"
+        data-showroom-title="title-1">
+      <a>Item 1</a>
+    </li>
+    <li class="item"
+        data-showroom-target="http://target.com/2"
+        data-showroom-title="title-2">
+      <a>Item 2</a>
+    </li>
+  </ul>
+
+.. code:: javascript
+
+  let items = document.querySelectorAll(".item");
+  let showroom = Showroom(items);
+
+or
+
+.. code:: javascript
+
+  let items = $(".item");
+  let showroom = Showroom(items);
+
+Configuration
+-------------
+
+The showrooms provides the following options.
+
++----------+--------------------------+--------------------------------------------------------------------------+
+| Option   | Default                  | Description                                                              |
++==========+==========================+==========================================================================+
+| cssClass | "ftw-showroom"           | Class attribute on the root element of the gallery                       |
++----------+--------------------------+--------------------------------------------------------------------------+
+| render   | See rendering_. section  | Override the default render behavior                                     |
++----------+--------------------------+--------------------------------------------------------------------------+
+| tail     | Empty function           | Is getting called when the user reaches the last element of the gallery  |
++----------+--------------------------+--------------------------------------------------------------------------+
+| head     | Empty function           | Is getting called when the user reaches the first element of the gallery |
++----------+--------------------------+--------------------------------------------------------------------------+
+| fetch    | See fetching_. section   | Override the default fetch behavior                                      |
++----------+--------------------------+--------------------------------------------------------------------------+
+| template | See template section     | Override the default gallery template                                    |
++----------+--------------------------+--------------------------------------------------------------------------+
+| target   | body Element             | Define a selector where the gallery is getting to be attached            |
++----------+--------------------------+--------------------------------------------------------------------------+
+
+.. _fetching:
+
+Fetching
+--------
+
+The default fetching function makes an AJAX call with the provided target on the item
```

Comment:
hmm this confuses me, plz rephrase, maybe:

The default fetching function uses the target provided by each item to make an AJAX call to retrieve its content.

?